.cicd/Jenkinsfile

```diff
@@ -122,17 +122,26 @@ pipeline {
                         }
                     }
 
-                    // Run the unified build script; if successful create a tarball of the build and upload to S3
-                    stage('Build') {
-                        steps {
-                            echo "Building SRW (${env.SRW_COMPILER}) on ${env.SRW_PLATFORM} (using ${env.WORKSPACE})"
-                            sh 'bash --login "${WORKSPACE}/.cicd/scripts/srw_build.sh"'
+                    parallel{
+                        // Run the unittest functional tests that require an HPC platform
+                        stage('Functional Tests) {
+                            steps {
+                                echo "Running unittest on retrieve_data.py"
+                                sh '${WORKSPACE}/.cicd/scripts/srw_unittest.sh"'
```

Comment:
On line 130, please add `bash --login` and `"`:

`sh 'bash --login "${WORKSPACE}/.cicd/scripts/srw_unittest.sh"'`